h/services/annotation_metadata.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+from sqlalchemy import func, select
+from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.orm import Session
+
+from h.models import Annotation, AnnotationMetadata, AnnotationSlim
+
+
+class AnnotationMetadataService:
+    def __init__(self, db: Session):
+        self._db = db
+
+    def set(self, annotation: Annotation, data: dict):
+        """Set the decrypted `jwe` dict as the metadata for `annotation`."""
```

Comment:
```suggestion
        """Upsert `data` into `annotation`'s metadata."""
```

The argument is no longer called `jwe`, and I don't think there's any encryption/decryption going on anymore? Also, I believe this merges the given `data` with the annotation's existing metadata (hence "upsert" instead of "set")?

Reply:
Ops, missed that :+1:

Reply:
>  this merges the given data with the annotation's existing metadata

No, this replaces the data with the one provided.